components/script/script_module.rs

```diff
@@ -229,12 +243,50 @@ impl ModuleTree {
         self.descendant_urls.borrow_mut().extend(descendant_urls);
     }
 
-    pub fn get_visited_urls(&self) -> &DomRefCell<HashSet<ServoUrl>> {
-        &self.visited_urls
+    fn recursive_check_descendants(
```

Comment:
Probably should have a doc comment saying that "this recursively checks if all of the transitive descendants are in the FetchingDescendants or later state"

Reply:
Thanks! Let me update!

Reply:
Fixed in 9abe9a8